pkg/pods/pod.go

```diff
@@ -229,11 +229,12 @@ func (pod *Pod) buildRunitServices(launchables []launch.Launchable, restartPolic
 				return util.Errorf("Duplicate executable %q for launchable %q", executable.Service.Name, launchable.ID())
 			}
 			sbTemplate[executable.Service.Name] = runit.ServiceTemplate{
+				Log: newManifest.GetLogExecutable(),
```

Comment:
So, this approach will require all pod manifests to specify their hook into the `log-bridge`. It seems like it might be an easier deployment story if we make the default log program something configured in the preparer's configuration and then just apply that to all pods.